UDV-Core/src/Modules/Temporal/Temporal.js

```diff
@@ -37,7 +37,15 @@ document.getElementById("temporal").innerHTML = '<button id="temporalTab">TEMPOR
 * @param options : optional parameters (including buildingVersions & Dates)
 */
 //=============================================================================
-function TemporalController(view, options={}) {
+/*
+Changer date en moment et casser le lien vers Vilo3D
+this.buildingDates deconne
+this.useBuildings
+this.buildingVersions
+this.syncBuildingVersionToCurrentDate()
```

Comment:
These comments might need to be removed from the code and moved to personal notes or into an issue on this repository.

Reply:
Comments placed in a new todo.md file.